Furball.Vixie/Graphics/Backends/Veldrid/FullScreenQuadShaders.cs

```diff
@@ -0,0 +1,47 @@
+namespace Furball.Vixie.Graphics.Backends.Veldrid {
+    internal class FullScreenQuadShaders {
+        public static string VertexShader = @"#version 450
```

Comment:
is there something preventing this from being in ShaderCode/Veldrid? If no i'd much prefer to have that there